core-bundle/src/Resources/contao/drivers/DC_Folder.php

```diff
@@ -53,6 +53,12 @@ class DC_Folder extends DataContainer implements \listable, \editable
 	 */
 	protected $strRootDir;
 
+	/**
+	 * Initial ID of the record
+	 * @var integer
+	 */
+	protected $initialId;
```

Comment:
Is it really really an integer? :upside_down_face:

Reply:
Probably as much as `$intId` is an integer in the `DC_Folder` class. 🙈

Reply:
Changed in bc42b3345f7ca3ea6f1df8d514af50f8ee66660d.